weblab/accounts/tests/test_signals.py

```diff
@@ -25,3 +27,25 @@ def test_email_sent_on_user_creation(superuser):
     assert 'Test User' in body
     assert 'test@example.com' in body
     assert 'http://127.0.0.1:8000/admin/accounts/user/{}/change/'.format(user.pk) in body
+
+
+@pytest.mark.django_db
```

Comment:
Sorry, should have noticed this before, but I think this test would be better in `accounts/tests/test_views.py` as it checks via the view rather than deleting the user directly.

The functionality of the test looks good though!